src/rv32_template.c

```diff
@@ -1861,7 +1861,7 @@ RVOP(
  * immediate, scaledby 4, to the stack pointer, x2, and writes the result to
  * rd'.
  * This instruction is used to generate pointers to stack-allocated variables,
- * and expands to addi rd', x2, nzuimm[9:2].
+ * and expands to addi rd', x2, nzimm[9:2].
```

Comment:
Quoting from riscv-spec-v2.2 [1]:
```
C.ADDI4SPN is a CIW-format RV32C/RV64C-only instruction that adds a zero-extended non-zero
immediate, scaled by 4, to the stack pointer, x2, and writes the result to rd0. This instruction is used
to generate pointers to stack-allocated variables, and expands to addi rd0, x2, nzuimm[9:2].
```
So, as far as I can see, there are no errors in the current description?

Link: https://riscv.org/wp-content/uploads/2017/05/riscv-spec-v2.2.pdf [1]

Reply:
I need some time to check.

Reply:
Thank you for the reminder, @visitorckw. I'll adjust my git commit. It's great to have you around.